human-viz-bridge/config/human-jointstate-bridge.ini

```diff
@@ -0,0 +1,25 @@
+## Module specific parameters
+name                           human-jointstate-bridge
+serverName                     human-state-provider 
+
+## URDF Model parameters
+urdf_model                     XSensURDF_subj1.urdf                    
+segmentList                    ("Pelvis", "L5", "L3", "T12", "T8","Neck","Head","RightShoulder","RightUpperArm","RightForeArm","RightHand","LeftShoulder","LeftUpperArm","LeftForeArm","LeftHand","RightUpperLeg","RightLowerLeg","RightFoot","RightToe","LeftUpperLeg","LeftLowerLeg","LeftFoot","LeftToe")
+jointList                      ("jL5S1_rotx", "jL5S1_roty", "jL4L3_rotx", "jL4L3_roty", "jL1T12_rotx", "jL1T12_roty", "jT9T8_rotx", "jT9T8_roty", "jT9T8_rotz",
+                               "jT1C7_rotx", "jT1C7_roty", "jT1C7_rotz", "jC1Head_rotx", "jC1Head_roty",
+                               "jRightC7Shoulder_rotx", "jRightShoulder_rotx", "jRightShoulder_roty", "jRightShoulder_rotz", "jRightElbow_roty", "jRightElbow_rotz", "jRightWrist_rotx", "jRightWrist_rotz",
+                               "jLeftC7Shoulder_rotx", "jLeftShoulder_rotx", "jLeftShoulder_roty", "jLeftShoulder_rotz", "jLeftElbow_roty", "jLeftElbow_rotz", "jLeftWrist_rotx", "jLeftWrist_rotz",
+                               "jRightHip_rotx", "jRightHip_roty", "jRightHip_rotz", "jRightKnee_roty", "jRightKnee_rotz", "jRightAnkle_rotx", "jRightAnkle_roty", "jRightAnkle_rotz", "jRightBallFoot_roty", 
+                               "jLeftHip_rotx", "jLeftHip_roty", "jLeftHip_rotz", "jLeftKnee_roty", "jLeftKnee_rotz", "jLeftAnkle_rotx", "jLeftAnkle_roty", "jLeftAnkle_rotz", "jLeftBallFoot_roty")
+
+## Connection parameters
+automaticConnection            true
+nodeName                       /human/joint_state_publisher
+jointStateTopicName            /human/joint_states
+tfTopicName                    /tf
+
+## RViz parameters
+worldRFName                    ground
+childLinkRFName                Pelvis
+
+                            
```

Comment:
some spaces to be removed 

All files should not have trailing spaces at the end of the lines (this means that each line should end with a "real" character or the newline)